.github/workflows/ci.yml

```diff
@@ -45,7 +45,7 @@ jobs:
   integration-tests:
     name: Integration Tests
     runs-on: ubuntu-20.04
-    container: golang:1.20-alpine3.18
+    container: golang:1.21-alpine3.18
```

Comment:
FYI - alpine newer than 3.18 do not support terraform (due to hashicorp licensing change). Consider switching to opentofu.